tests/unit/scripting.tcl

```diff
@@ -1411,6 +1411,9 @@ start_server {tags {"scripting"}} {
             r eval {return redis.call('set','x','y')} 1 x
         }
         assert_equal [errorrstat OOM r] {count=1}
+        assert_equal [totalerrorreplies r] {1}
+        assert_match {*cmdstat_set:calls=0*rejected_calls=1,failed_calls=0*} [r info commandstats]
+        assert_match {*cmdstat_eval:calls=1*rejected_calls=0,failed_calls=1*} [r info commandstats]
```

Comment:
that's week (the `*` can be applied on several lines). should use something like this, to match just one command:
```
assert_match {*calls=1,*,rejected_calls=0,failed_calls=1} [cmdrstat hgetall r]
```

Reply:
Fixed